### PR TITLE
Add some metrics to Coprocessor

### PR DESCRIPTION
**Pull request by breeswish**
## What have you changed? (mandatory)

- Added vectorization metrics
- Added response size metrics

## What are the type of the changes? (mandatory)

- Improvement (change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

Ansible will be updated later.

### CODE REVIEW

File: src/coprocessor/metrics.rs [hunk 1]
```diff
@@ -56,6 +56,17 @@ lazy_static! {
         &["req", "metric"]
     )
     .unwrap();
+    pub static ref COPR_DAG_REQ_COUNT: IntCounterVec = register_int_counter_vec!(
+        "tikv_coprocessor_dag_request_count",
+        "Total number of DAG requests",
+        &["vec_type"]
+    )
+    .unwrap();
+    pub static ref COPR_RESP_SIZE: IntCounter = register_int_counter!(
```
Comment (sticnarf) [inline src/coprocessor/metrics.rs]: Is simply a counter enough? Do we need a histogram?

Comment (breeswish) [reply src/coprocessor/metrics.rs]: We mostly use it to observe whether or not Coprocessor is the cause of OOM, so a Counter that includes byte sizes of all requests is enough.

Comment (sticnarf) [reply src/coprocessor/metrics.rs]: Got it. Then, for OOM detection, is it better to add metrics of bytes already sent by gRPC (after sink sending finished)?

Comment (breeswish) [reply src/coprocessor/metrics.rs]: Yes. This metrics will be added later by someone else.

Comment (breeswish) [reply src/coprocessor/metrics.rs]: Currently we are able to recognize Coprocessor OOM by looking at metrics in this PR + Network outbound metrics.

**Pull request merged**